src/psfmachine/machine.py

```diff
@@ -783,6 +793,12 @@ def _time_bin(self, npoints=200, downsample=False):
                 )
             pc1_smooth = np.array(pc1_smooth)
             pc2_smooth = np.array(pc2_smooth)
+            pc1_smooth = (pc1_smooth - pc1_smooth.mean()) / (
```

Comment:
Is this just normalization?

Reply:
yes, that standardize the poscorrs to be zero-centered